eng/Versions.props

```diff
@@ -82,5 +82,19 @@
     Versions below this comment are not managed by automation and can be changed as needed.
   -->
   <PropertyGroup Label="Manual">
+    <!-- Compatibility with VS 17.4/.NET SDK 7.0.1xx -->
+    <MicrosoftCodeAnalysisVersion_4_4>4.4.0</MicrosoftCodeAnalysisVersion_4_4>
+    <!-- Compatibility with VS 17.8/.NET SDK 8.0.1xx -->
+    <MicrosoftCodeAnalysisVersion_4_8>4.8.0</MicrosoftCodeAnalysisVersion_4_8>
```

Comment:
How is this property used? Why do we need it?

Reply:
We only have been targeting Roslyn 4.4 where as dotnet/runtime also has targeting projects for 3.11 and 4.0 and 4.4+. This part is being added to prep in case we want to use the same approach. Right now it's still using only 4.4

Reply:
If we don't use it - then we shouldn't have it. We should only add it when/if it's required. Otherwise, it, essentially, becomes a technical debt.